src/app/modules/project/edit/edit.component.html

```diff
@@ -94,12 +93,12 @@ <h3>Contributor function</h3>
     <div class="row" *ngIf="collaborators.length > 0">
 
       <div class="col-12">
-        <h3>Contributors</h3>
+        <h3>Collaborators</h3>
       </div>
 
       <div *ngFor="let collaborator of collaborators" class="col-5">
         <button type="button" class="btn btn-icon btn-danger" aria-label="error"
-          (click)="onClickDeleteContributor(collaborator)">
+          (click)="onClickDeleteContributor(collaborator)"> <span aria-hidden="true"> &times; </span>
```

Comment:
If we are renaming all contributors strings to collaborators, I suggest we also rename the method to reflect the same changes

Reply:
Agreed, thanks. Done!